content-test/selectors/selectors.test.js

```diff
@@ -93,6 +96,14 @@ describe("selectors", () => {
         images: []
       });
     });
+    it("should remove urls in block list", () => {
+      let frecent = fakeState.FrecentHistory.rows.splice(0,3);
```

Comment:
I'm a bit surprised ESLint didn't throw a warning about the lack of a space after the comma.
Either we don't have an error level set high enough, or haven't enabled the correct rule, or maybe not fully linting the /content-test/ dir.